src/chi/javascript/components/mobile-navigation.js

```diff
@@ -0,0 +1,458 @@
+import { Component } from '../core/component';
+import { Util } from '../core/util.js';
+import { chi } from '../core/chi';
+import { Drawer } from './drawer';
+import { Dropdown } from './dropdown';
+
+const CHI_TABS_CLASS = 'chi-tabs';
```

Comment:
Do you think it makes sense to create a separate file for these constants?

Reply:
`src/javascript/core/chi.js` is a file with general chi.js classes used in multiple components.
At some point, we will have to do a major refactoring of all the chi.js components and create separate files for component-specific classes but for now, I think we can leave it as it is.
Thanks!